### PR TITLE
Add stuff from UvA and Radboud University

### PR DESCRIPTION
**Pull request by ZhengyuZhao**
Add Marcel Worring and Arnold W. M. Smeulders from University of Amsterdam

# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

_Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [x] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -8967,6 +8997,7 @@ Jon Whittle,Monash University,https://research.monash.edu/en/persons/jon-whittle
 Jon Youn,University of Nebraska - Omaha,https://www.cune.edu,NOSCHOLARPAGE
 Jonas Almström Duregård,Chalmers/GU,https://www.chalmers.se/en/staff/Pages/jonas-duregard.aspx,NOSCHOLARPAGE
 Jonas Kuhn,University of Stuttgart,http://www.ims.uni-stuttgart.de/institut/mitarbeiter/jonas,t5vqVEkAAAAJ
+Jonas Teuwen,Radboud University,http://diagnijmegen.nl/index.php/Person?name=Jonas_Teuwen,NOSCHOLARPAGE
```
Comment (emeryberger) [inline csrankings.csv]: This person does not appear to be in CS.

File: csrankings.csv [hunk 2]
```diff
@@ -8327,6 +8355,7 @@ Jeroen J. A. Keiren,TU Eindhoven,https://www.jeroenkeiren.nl,_nrYKCoAAAAJ
 Jeroen Keiren,TU Eindhoven,https://www.jeroenkeiren.nl,_nrYKCoAAAAJ
 Jeroen Keppens,King's College London,https://www.kcl.ac.uk/people/jeroen-keppens,6uEtmfoAAAAJ
 Jeroen van de Graaf,UFMG,http://homepages.dcc.ufmg.br/~jvdg,-w8olWwAAAAJ
+Jeroen van der Laak,Radboud University,http://diagnijmegen.nl/index.php/Person?name=Jeroen_van_der_Laak,suzUVuwAAAAJ
```
Comment (emeryberger) [inline csrankings.csv]: Not in CS.

File: csrankings.csv [hunk 3]
```diff
@@ -6983,6 +7006,8 @@ Heng Wang,Peking University,http://eecs.pku.edu.cn/EN/People/Faculty/Detail/?ID=
 Heng Yin,University of California - Riverside,http://www.cs.ucr.edu/~heng,_1VlI00AAAAJ
 Heng-Da Cheng,Utah State University,https://directory.usu.edu/aggie/152b1,NOSCHOLARPAGE
 Heni Ben Amor,Arizona State University,https://cidse.engineering.asu.edu/directory/ben-amor-heni,u_7S7VYAAAAJ
+Henk van den Heuvel,Radboud University,https://www.ru.nl/english/people/heuvel-h-van-den/,F5j9Ud4AAAAJ
+Henkjan J. Huisman,Radboud University,http://diagnijmegen.nl/index.php/Person?name=Henkjan_Huisman,zkMmvCwAAAAJ
```
Comment (emeryberger) [inline csrankings.csv]: Not in CS.

File: csrankings.csv [hunk 4]
```diff
@@ -6961,6 +6983,7 @@ Helge Rhodin,University of British Columbia,http://helge.rhodin.de,2maWWboAAAAJ
 Helger Lipmaa,University of Tartu,http://kodu.ut.ee/~lipmaa,kZch-sMAAAAJ
 Helle Hein,University of Tartu,http://www.ut.ee/en/kontakt/arvutiteaduse-instituut,NOSCHOLARPAGE
 Helle Rootzén,DTU,https://www.compute.dtu.dk/english,NOSCHOLARPAGE
+Helmer Strik,Radboud University,https://www.ru.nl/english/people/strik-w/,PuvTkHAAAAAJ
```
Comment (emeryberger) [inline csrankings.csv]: Not in CS.

**Review by emeryberger — CHANGES_REQUESTED** [review]
My spot check turned up a number of faculty who are not in a CS department, raising the question of their eligibility for inclusion. Please examine the inclusion criteria and address this, specifically the requirement that they be able to _solely_ advise a PhD student _in CS_.

File: csrankings.csv (hunk 1 above)
Comment (ZhengyuZhao) [reply csrankings.csv]: > This person does not appear to be in CS.
Thanks for your checking. Actually, at Radboud University, we have an organization called Radboud AI, which involves researchers from different sections, such as cognitive neural science, machine learning, medical imaging, language technology...(more details in https://www.ru.nl/ai/research-education/research-groups/). Evidently, most researchers I updated have already published papers in top AI conferences such as NeurIPS, ACL, SIGIR, etc.. But you are right that the people you have spotted have no publications in those conferences so far. I suppose they supervise PhD students in the related topics, but not sure at this moment. You can remove them so.

Comment (emeryberger) [reply csrankings.csv]: Please remove these faculty - thanks.

Comment (emeryberger) [reply csrankings.csv]: Note that pubs in the conferences aren't the deciding factor - it's whether or not they can solely supervise PhDs in computer science.

Comment (ZhengyuZhao) [reply csrankings.csv]: > Please remove these faculty - thanks.

Hi, I just removed 9 people (from medical areas) who are not eligible with respect to the policies as you pointed out. The rest are OK now.

**Comment by emeryberger** [comment]
You need to resolve conflicts and try again.

**Comment by ZhengyuZhao** [comment]
> You need to resolve conflicts and try again.

Resolved!

**Pull request merged**